mock/docs/site-defaults.cfg

```diff
@@ -498,7 +498,16 @@
 # the whole /etc/pki/ca-trust/extracted directory recursively by default, and
 # Fedora or EL systems work with that.  But some destination chroots can have
 # different configuration, and copying the bundle helps.
-#config_opts['ssl_ca_bundle_path'] = None
+# config_opts['ssl_ca_bundle_path'] = None
+
+# Copy host's SSL certificates into a specified location inside the chroot if
+# mock needs access to repositories which require client certificate
+# authentication. Specify the full path to the public certificate on the host
+# and the destination directory in the chroot. Do the same for the private key.
+# The private key should not be password-protected if you want mock to run
+# unattended.
+# config_opts['ssl_extra_certs'] = ['/etc/pki/tls/certs/client.crt', '/etc/pki/tls/certs/',
+#                                   '/etc/pki/tls/private/client_nopass.key.crt', '/etc/pki/tls/private/']
```

Comment:
The values in site=defaults.cfg are the defaults. But the default for ssl_extra_cert is None.
This example is fine if you clearly mark it as example **and** state the default.

Reply:
Thank you for pointing that out. Fixed.

Reply:
I know it is a matter of taste, but this config option could easily be defined as a list of pairs, or a dict with {"source": "dest"}

Reply:
Given my limited pyFoo I just followed def copy_certs(self) so no taste was involved, just the quickest path to a solution :-)